### PR TITLE
negar/password_entropy

### PR DESCRIPTION
**Pull request by negar-binary**
password entropy

### CODE REVIEW

File: src/javascript/binary/base/passwordmeter.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+       strVerdict = text.localize("Strong");
+    }
+    else
+    {
+       strVerdict = text.localize("Stronger");
+    }
+
+  var array = [intScore, strVerdict];
+  return array;
+}
```
Comment (borisyankov) [inline src/javascript/binary/base/passwordmeter.js]: That lib doesn't look very good, to be honest.
We had discussed, and I proposed that actually good one:
https://github.com/dropbox/zxcvbn

Based on:
https://xkcd.com/936/

:)

Comment (negar-binary) [inline src/javascript/binary/base/passwordmeter.js]: @boeisyankov that lib was rejected by JY and this was the lib he asked me to use

Comment (borisyankov) [inline src/javascript/binary/base/passwordmeter.js]: Why?

Comment (borisyankov) [inline src/javascript/binary/base/passwordmeter.js]: zxcvbn is a library created and used by Dropbox - a $10b company.
This one is 10 year old 'quick and dirty password quality meter' - by the words of the authors.
Its calculations give completely incorrect results of the actual entropy.

**Comment by mwmiller** [comment]
You probably want something which actually [computes entropy](https://github.com/mvhenten/string-entropy) instead of an ad-hoc collection of heuristics.

You could wrap the library fairly simply to return score and verdict like the above.

**Comment by negar-binary** [comment]
@borisyankov @mwmiller please confirm with JY as he is the one who chose this library

**Pull request closed without merging**